textalyze.js

```diff
@@ -1,10 +1,12 @@
+const fs = require('fs');
+
 /**
  * Given an input String, returns true if it's a valid string false otherwise
```

Comment:
Small English nitpick: 

> Given an input String, returns true if it's a valid string false otherwise

I would say:

> Given an input object, returns true of the input is a string and false otherwise.

The reasoning is two-fold: 
1. The input argument may or may not be a string
1. "Valid" sounds like we're running a check against the string, that is, some strings are valid and others aren't.  But really *all* strings are good and *all* non-strings are bad.